pkgs/top-level/python-packages.nix

```
@@ -4636,6 +4636,8 @@ self: super: with self; {
 
   hexdump = callPackage ../development/python-modules/hexdump { };
 
+  hfst-python = callPackage ../development/python-modules/hfst-python { };
```

```suggestion
  hfst = callPackage ../development/python-modules/hfst { };
```

the pypi name is without -python https://pypi.org/project/hfst/